.github/workflows/unittests-gpu.yml

```diff
@@ -79,13 +79,19 @@ jobs:
           cat jobid.log | xargs -i aws s3api wait object-exists --bucket gluon-nlp-dev --key batch/{}/temp/coverage.xml
           cat jobid.log | xargs -i aws s3 cp s3://gluon-nlp-dev/batch/{}/temp/coverage.xml ./coverage.xml
 
-      - name: Upload coverage to Codecov
-        if: ${{ failure() || success() }}
+      - name: Upload coverage to Codecov(For pull request)
```

Comment:
Why create two separate steps? You can also use an if else statement inside the step. It's just bash code